cadence/contracts/bridge/FlowEVMBridgeHandlerInterfaces.cdc

```diff
@@ -0,0 +1,132 @@
+import "FungibleToken"
+import "NonFungibleToken"
+
+import "EVM"
+
+/// FlowEVMBridgeHandlerInterfaces
+///
+/// This contract defines the interfaces for the FlowEVM Bridge Handlers. These Handlers are intended to encapsulate
+/// the logic for bridging edge case assets between Cadence and EVM and require configuration by the bridge account to
+/// enable. Contracts implementing these resources should be deployed to the bridge account so that privileged methods,
```

Comment:
It sounds to me like you're saying here that a contract like `FiatToken` would have to be deployed to the bridge account to be compatible. Is that true? I don't think that will be possible since these are already deployed to other accounts

Reply:
Good question. No, the existence of handler assumes the contract is not deployed to the bridge. Rather, what I mean by `Contracts implementing these resources should be deployed to the bridge account` is that contracts implementing the `*Handler*` interfaces should be deployed to the bridge. Concretely, `FlowEVMBridgeHandlers` is one such contract.